edweiss-app/app/(app)/courses/[id]/index.tsx

```diff
@@ -313,7 +332,7 @@ const CoursePage: ApplicationRoute = () => {
 				</For>
 
 				{/* Bouton vers les Passed Assignments */}
```

Comment:
A nitpick: comments should be in english

Reply:
Yes! Thank you